### DISCUSSION TITLE
Cleanup Job Pods still existed after completing the job

### DISCUSSION THREAD

**Discussion by alionar** [apache/airflow#26547] [Q&A] [answered]
<img width="919" alt="Screen Shot 2022-09-21 at 10 36 29" src="https://user-images.githubusercontent.com/18596757/191408780-fa3e0aba-2faf-45d2-b0ca-6c8c8db458d2.png">

Airflow Version: 2.2.2
Kubernetes Version: 1.22.12-gke.500
Helm Chart version: 1.6.0

Hi, i found that completed cleanup job pods just stayed in nodes after finished and make GKE auto scaller triggered to add new nodes everytime cleanup job pods executed.

It made us to delete all completed job pods and drain the unused nodes manually everyday.

I found in k8s docs that:
```
The .spec.successfulJobsHistoryLimit and .spec.failedJobsHistoryLimit fields are optional. 
These fields specify how many completed and failed jobs should be kept. 
By default, they are set to 3 and 1 respectively
```

But when i tried to add this history limit as 0 into cleanup helm chart value, chart didn't recognize it
![Screen Shot 2022-09-21 at 10 40 19](https://user-images.githubusercontent.com/18596757/191409229-ff2336df-c417-4e70-b9cd-f837b053f3b5.png)

is that possible to add some job history limit number in cleanup value chart?

Current values.yaml for cleanup:
```
cleanup:
  enabled: true
  schedule: "*/15 * * * *"
```

**Comment by potiuk** [comment, marked as answer]
created an issue https://github.com/apache/airflow/issues/26709 from that. - seems we have no such option, Marked it as good first issue, naybe someone will pick it and implement it (or you could if you would like to @alionar )

**Reply by alionar** [reply]
I appreciate the assistance @potiuk 👍 

Btw, just checked the issue. It already merged for helm chart 1.7.0 release
